<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recipeExtrasDialog</class>
 <widget class="QDialog" name="recipeExtrasDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recipe Extras</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Brewer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_brewer">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Asst. Brewer</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Taste Rating</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinBox_tasteRating">
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::UpDownArrows</enum>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_asstBrewer">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Primary Age (days)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_primaryAge">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Primary Temp</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="lineEdit_primaryTemp">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Secondary Age (days)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="lineEdit_secAge">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Secondary Temp</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="lineEdit_secTemp">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Tertiary Age (days)</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lineEdit_tertAge">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Tertiary Temp</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="lineEdit_tertTemp">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Bottle/Keg Age (days)</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="lineEdit_age">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Bottle/Keg Temp</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="lineEdit_ageTemp">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Date First Brewed</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QDateEdit" name="dateEdit_date">
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="displayFormat">
          <string>dd MMM yyyy</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Carbonation Volumes</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLineEdit" name="lineEdit_carbVols">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Taste Notes</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="BtTextEdit" name="btTextEdit_tasteNotes">
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Notes</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="BtTextEdit" name="btTextEdit_notes">
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_save">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../build/brewtarget.qrc">
         <normaloff>:/images/filesave.svg</normaloff>:/images/filesave.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BtTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>BtTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit_brewer</tabstop>
  <tabstop>lineEdit_asstBrewer</tabstop>
  <tabstop>spinBox_tasteRating</tabstop>
  <tabstop>lineEdit_primaryAge</tabstop>
  <tabstop>lineEdit_primaryTemp</tabstop>
  <tabstop>lineEdit_secAge</tabstop>
  <tabstop>lineEdit_secTemp</tabstop>
  <tabstop>lineEdit_tertAge</tabstop>
  <tabstop>lineEdit_tertTemp</tabstop>
  <tabstop>lineEdit_age</tabstop>
  <tabstop>lineEdit_ageTemp</tabstop>
  <tabstop>dateEdit_date</tabstop>
  <tabstop>lineEdit_carbVols</tabstop>
  <tabstop>btTextEdit_tasteNotes</tabstop>
  <tabstop>btTextEdit_notes</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_cancel</tabstop>
 </tabstops>
 <resources>
  <include location="../build/brewtarget.qrc"/>
 </resources>
 <connections/>
</ui>
